server/src/controllers/planterRegistration.controller.ts

```diff
@@ -35,7 +35,7 @@ export class PlanterRegistrationController {
     return await this.planterRepository.find(filter);
   }
 
-  @get('/planter/{id}', {
+  @get('/planter_registration/{id}', {
```

Comment:
I don't know how many other urls we use, but I think it's preferable to use a dash instead of an underscore
For SEO as an example: https://developers.google.com/search/docs/advanced/guidelines/url-structure?hl=en&visit_id=637427852299866372-1076474063&rd=1

Reply:
I agree. This was originally done for consistency with existing urls, but I'm inclined to move them all away from underscores.
I'll fix this one here and check with the team before changing the rest.